forc-pkg/src/manifest.rs

```diff
@@ -223,6 +223,9 @@ pub struct BuildProfile {
     pub json_abi_with_callpaths: bool,
     #[serde(default)]
     pub error_on_warnings: bool,
+    #[serde(default)]
+    pub reverse_results: bool,
+    pub experimental_private_modules: bool,
```

Comment:
I think this field `experimental_private_modules` was removed at some point as it isn't on `master` anymore. I also think the `serde` annotation above `reverse_results` was intended for that field and can be removed as well. cc @kayagokalp

Reply:
Ah I see. Fixing :)